packages/tupaia-web/src/api/queries/useMapOverlays.ts

```diff
@@ -62,6 +64,8 @@ export const useMapOverlays = (): UseMapOverlaysResult => {
 
   const updateSelectedMapOverlay = (e: ChangeEvent<HTMLInputElement>) => {
```

Comment:
This is a bit pedantic and academic but I reckon that the updateSelectedMapOverlay should probably be moved out of this hook and into a util as well to keep the data fetching a bit cleaner.